Makefile

```diff
@@ -0,0 +1,25 @@
+CONTAINER_REPO ?= ghcr.io/changeme-org/changeme-gadget-name
+IMAGE_TAG ?= latest
+
+CLANG_FORMAT ?= clang-format
+
+.PHONY: build
+build:
+	IG_EXPERIMENTAL=true sudo -E \
+		ig image build \
+			-t $(CONTAINER_REPO):$(IMAGE_TAG) \
+			--update-metadata .
```

Comment:
Why always using `--update-metadata`? 

Not related to this template but IG framework: I'm wondering if we should enable this feature by default.

Reply:
I think this is an issue, it should update it with `changeme_mygadget` from the eBPF code. Let me check this upstream.